jobs/get_all_cards_gold.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
Can you change the name of the script to ajitava_cards_gold.py

Reply:
Done.